doop.c

```diff
@@ -635,6 +635,37 @@ Perl_do_trans(pTHX_ SV *sv)
     }
 }
 
+static int
```

Comment:
that should be STATIC.

Reply:
Also actually why is it "int"? this is returning a bool it should be typed bool.